namerd/storage/k8s/src/main/scala/io/buoyant/namerd/storage/K8sV2DtabStoreInitializer.scala

```diff
@@ -0,0 +1,28 @@
+package io.buoyant.namerd.storage
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import io.buoyant.config.types.Port
+import io.buoyant.k8s.ClientConfig
+import io.buoyant.namerd.{DtabStore, DtabStoreConfig, DtabStoreInitializer}
+
+case class K8sConfigV2(
```

Comment:
I'm not a fan of this naming. I don't think `V2` is very descriptive – I think the class name should make it clear that the `DtabStore` created by this config is the CRD-based dtab store. Consider `K8sCrdConfig`?

Reply:
Thats a good point. I was thinking that we calling it `K8sCrdConfig` would give details about the underlying implementation of the k8s storage plugin but now that I think of it. This is an open source project. :facepalm:

Reply:
I think whether CRDs or TPRs are used reflects both the implementation of the object and its _purpose_.